src/components/CommentInfo.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { Comment } from '../types/Comment';
+
+type Props = {
+  comment: Comment[],
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => (
+  <>
+    {comment.map(com => (
```

Comment:
move mapping to CommentList component, CommentInfo will be responsible for receiving one comment and rendering it